web/crux-ui/src/pages/nodes.tsx

```diff
@@ -96,7 +109,36 @@ const NodesPage = (props: NodesPageProps) => {
       {!creating ? null : <EditNodeCard className="mb-2" submitRef={submitRef} onNodeEdited={onCreated} />}
       {filters.items.length ? (
         <>
-          <Filters setTextFilter={it => filters.setFilter({ text: it })} />
+          <DyoCard className="flex flex-col p-8">
```

Comment:
Wouldn't the Filters component with DyoChips inside be better? Like on the products page